scripts/glucose_CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ set_target_properties(
     XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "Developer ID Application: Daniel Kroening"
 )
 
+if(ipo_supported)
+    set_property(TARGET minisat2-condensed PROPERTY INTERPROCEDURAL_OPTIMIZATION True)
```

Comment:
Copy&paste error: Should be "glucose-condensed"

Reply:
oops, fixed

Reply:
Looks like this needs to be fixed again.